javatests/arcs/core/storage/handle/SingletonIntegrationTest.kt

```diff
@@ -120,6 +120,19 @@ class SingletonIntegrationTest {
         assertThat(singletonB.fetch()).isNull()
     }
 
+    @Test
+    fun syncsBeforeClearing() = runBlockingTest {
```

Comment:
a better name for the test would "canClearValueFromOtherActor" or something like that, since that is what you are verifying.

Reply:
Went for something similar to what you suggested, but more in line with how the other test cases are named